content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -594,6 +594,13 @@ appropriate arguments.
 More options and information about the
 [`kubeadm reset command`](/docs/reference/setup-tools/kubeadm/kubeadm-reset/).
 
+## Self-hosting the control plane of a cluster
+
+Self-hosting the control plane is no longer a supported configuration. While kubeadm will still pivot your cluster
+to an self hosted one, the cluster becomes un-supported by kubeadm.
+
+documentation can be found [here.](/docs/reference/setup/independent/self-hosting)
```

Comment:
```suggestion
[Documentation](/docs/reference/setup/independent/self-hosting) on the deprecated self-hosting feature.
```

No strong opinion. I have a slight preference to avoid “here” as the text of a hyperlink.

Reply:
location changed per @neolit123 's comment, but updated the style to above. thank you!

Reply:
Not sure if you elided part of “Documentation ~on the deprecated self-hosting feature~” intentionally or because GitHub made it hard to spot @Klaven.

Either's fine by me; again, no strong opinion.

Reply:
Thanks for updating.